src/zope/app/publisher/xmlrpc/README.rst

```diff
@@ -244,7 +244,7 @@ If you need to raise an error, the prefered way to do it is via an
   ...         self.request = request
   ...
   ...     def your_fault(self):
-  ...         return xmlrpclib.Fault(42, "It's your fault!")
+  ...         return xmlrpclib.Fault(42, "It's your fault 😢!")
```

Comment:
Could this be a unicode escape instead of a high-byte embedded character? Especially since there's no standard way to declare an encoding for .rst files in the file, I worry about it getting mangled by editors or something else. (Plus, doesn't this need a unicode prefix for Python 2?)
```suggestion
  ...         return xmlrpclib.Fault(42, u"It's your fault \N{CRYING FACE}!")
```

(Except that CRYING FACE is relatively new and might not be in the unicode data a version was built with; SNOWMAN should be, I think.)

Reply:
Okay, convinced, I am going to use the ☃.